ansible_collections/arista/avd/molecule/eos_designs-mpls-isis-sr-ldp/intended/structured_configs/SITE2-LSR2.yml

```diff
@@ -152,5 +174,30 @@ port_channel_interfaces:
     ldp:
       interface: true
       igp_sync: true
+- name: Port-Channel110
+  peer: SITE2-LER1
+  peer_interface: Port-Channel13
```

Comment:
Seems like the peer channel ID is still following the old behavior. This also affect the description. Either we should have a list of IDs so you can control each end seperately, or we should use the given ID for both ends.

Reply:
Thank you @ClausHolbechArista  - good catch. I've attempted to fix it in another commit, please let me know if you like the implementation though.